src/Bex/Behat/ScreenshotExtension/Event/ScreenshotUploaderEvent.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Bex\Behat\ScreenshotExtension\Event;
+
+use Symfony\Contracts\EventDispatcher\Event;
+
+/**
+ * Dispatches an event when screenshots are uploaded.
+ */
+class ScreenshotUploaderEvent extends Event
+{
+    public const UPLOAD = 'screenshot-uploader-upload';
```

Comment:
This constant should hold the name of the event for configuration purposes. So instead of calling it `UPLOAD`, I would change it to `NAME` like how it's demonstrated in the [Symfony documentation](https://symfony.com/doc/current/components/event_dispatcher.html#creating-an-event-class).
The value is a configuration key name where dots can be used as namespace separators (see [naming conventions](https://symfony.com/doc/current/components/event_dispatcher.html#naming-conventions)). So it would make it to be like:
``` php
const NAME = 'screenshot.uploader.uploaded';
const NAME = 'screenshot.uploader.upload_complete'; // alternatively
```